content/en/docs/setup/production-environment/turnkey/aws.md

```diff
@@ -18,7 +18,7 @@ To create a Kubernetes cluster on AWS, you will need an Access Key ID and a Secr
 
 ### Supported Production Grade Tools
 
-* [conjure-up](/docs/getting-started-guides/ubuntu/) is an open-source installer for Kubernetes that creates Kubernetes clusters with native AWS integrations on Ubuntu.
+* [conjure-up](https://conjure-up.io/) is an open-source installer for Kubernetes that creates Kubernetes clusters with native AWS integrations on Ubuntu.
```

Comment:
@makocchi-git , the change seems fine. This is a link to a third party tool (Canonical).
What do you think about linking directly to the Kubernetes/AWS integration setup page instead of the main page of `conjure-up`?
https://docs.conjure-up.io/stable/en/cni/k8s-and-aws

Reply:
@kbhawkey Thank you for your comments.
I thought that this deep link has a little bit risky for the location changing by Canonical.
This is the reason why I linked the top-level page.
However, as you mentioned, `docs.confure-up.io/..` page is more useful for readers than the top-level page.

So I will change the link to `docs.confure-up.io/...`.

Reply:
Thanks!
/lgtm